protocol/protobuf/proto/state_representation/space/cartesian/cartesian_state.proto

```diff
@@ -0,0 +1,46 @@
+syntax = "proto3";
+
+package state_representation.proto;
+
+import "state_representation/space/spatial_state.proto";
+
+message Vector3d {
+  double x = 1;
+  double y = 2;
+  double z = 3;
+}
+
+message Quaterniond {
+  double w = 1;
+  Vector3d vec = 2;
+}
+
+message CartesianState {
+  SpatialState spatial_state = 1;
+  Vector3d position = 2;
+  Quaterniond orientation = 3;
+  Vector3d linear_velocity = 4;
+  Vector3d angular_velocity = 5;
+  Vector3d linear_acceleration = 6;
+  Vector3d angular_acceleration = 7;
+  Vector3d force = 8;
+  Vector3d torque = 9;
+}
+
+message CartesianPose {
+  SpatialState spatial_state = 1;
+  Vector3d position = 2;
+  Quaterniond orientation = 3;
+}
+
+message CartesianTwist {
+  SpatialState spatial_state = 1;
+  Vector3d linear_velocity = 4;
```

Comment:
Are those fields number 4 and 5 on purpose (if so why) or is it from copying? same in CartesianWrench

Reply:
great catch! thank you for looking thoroughly despite there being so many files. Since CartesianTwist and Wrench are unique messages, they also have unique fields and so should maintain the standard numbering. This is a copying error from CartesianState